### PR TITLE
upgrade Junit

### PR DESCRIPTION
**Pull request by aman-sai**
- Upgraded to Junit

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -232,6 +233,41 @@
                         <goals>
                             <goal>report</goal>
                         </goals>
+                        <configuration>
+                            <haltOnFailure>false</haltOnFailure>
```
Comment (r0goyal) [inline pom.xml:237]: @aman-sai Why are these changes required ?

**Comment by r0goyal** [comment]
This MR now only upgrades JUnit 5. Change description accordingly

File: pom.xml (hunk 1 above)
Comment (aman-sai) [reply pom.xml]: I've written it when the repo was using cobertura. Now it's using Jacoco. we don't need these changes now

**Review by r0goyal — APPROVED** [review]

**Review by santanusinha — APPROVED** [review]
